react-carousel-component/my-app/src/Carousel.js

```diff
@@ -0,0 +1,66 @@
+import CircleButton from './CircleButton';
+import ArrowButton from './ArrowButton';
+import CarouselImage from './CarouselImage';
+import './Carousel.css';
+import { useState, useEffect, useCallback } from 'react';
+
+export default function Carousel({ itemsArray }) {
+  const [bannerIndex, setBannerIndex] = useState(0);
+
+  /**
+   * I created an increment callback function so the 'useEffect' looks cleaner. Otherwise,
+   * I would just have the same setBannerIndex call inside useEffect and the dependencies
+   * would be bannerIndex and itemsArray.length, instead of incrementCB. I wasn't sure
+   * which was more 'industry standard'
```

Comment:
What you've done is great!

I'm not thrilled by the `incrementCB` name, though. I like names to indicate what they do or are used for instead of how they are used. I would have preferred `incrementIndex` or `incrementBanner` or `nextImage`.